dash_eee/pages/warehouse.py

```diff
@@ -0,0 +1,318 @@
+                html.P(
+                    "Some quick example text to build on the card title and "
+                    "make up the bulk of the card's content.",
+                    className="card-text",
+                ),
+                dcc.Dropdown(
+                    id='genders3',
+                    options=choice_list_gender,
+                    style={
+                "background": "#8cd98c",
+            },
+                    value="Both",
+                    clearable=False
+                ),
+            dcc.Graph(id='example-graph5')
+            ]
+        ),
+    ],
+    style={"width": "29rem", "height":'40rem'},
+)
+
+card6 = dbc.Card(
+    [
+        dbc.CardBody(
+            [
+                html.H4("Card title", className="card-title"),
+                html.P(
+                    "Some quick example text to build on the card title and "
+                    "make up the bulk of the card's content.",
+                    className="card-text",
+                ),
+                dcc.Dropdown(
+                    id='genders4',
+                    options=choice_list_gender,
+                    style={
+                "background": "#8cd98c",
+            },
+                    value="Both",
+                    clearable=False
+                ),
+            dcc.Graph(id='example-graph6')
+            ]
+        ),
+    ],
+    style={"width": "29rem", "height":'40rem'},
+)
+
+card7 = dbc.Card(
+    [
+        dbc.CardBody([
+            html.H4("Card title", className="card-title"),
+            html.P(
+            "Some quick example text to build on the card title and "
+            "make up the bulk of the card's content.",
+            className="card-text",
+        ),
+        dcc.Graph(
+                id='delivery-map',
+                figure=fig7,
+                style={"height": "28rem"},
+            )
+    ])
+    ],
+    style={"width": "29rem", "height": "40rem"},
+)
+
+card8 = dbc.Card(
+    [
+        dbc.CardBody(
+            [
+                html.H4("Card title", className="card-title"),
+                html.P(
+                    "Some quick example text to build on the card title and "
+                    "make up the bulk of the card's content.",
+                    className="card-text",
+                ),
+            dcc.Graph(id='example-graph7')
+            ]
+        ),
+    ],
+    style={"width": "29rem", "height":'40rem'},
+)
+
+card9 = dbc.Card(
+    [
+        dbc.CardBody(
+            [
+                html.H4("Card title", className="card-title"),
+                html.P(
+                    "Some quick example text to build on the card title and "
+                    "make up the bulk of the card's content.",
+                    className="card-text",
+                ),
+            dcc.Graph(id='example-graph8')
+            ]
+        ),
+    ],
+    style={"width": "29rem", "height":'40rem'},
+)
+
```

Comment:
The cards use a lot of the same code so maybe I could make a function which can initialise a card and set the figure.